utils/onboarding/wait_for_tcp_port.py

```diff
@@ -19,6 +19,7 @@
 
 import socket
 import time
+from utils.tools import logger
 
 
 def wait_for_port(port: int, host: str = "localhost", timeout: float = 5.0):
```

Comment:
Probaly a better name : `wait_for_accepting_connection`

Reply:
for the next pr